PlayFabClientSDK/source/PlayFabHttp/PlayFabWinHttp.cs

```diff
@@ -11,20 +10,10 @@ namespace PlayFab.Internal
     public class PlayFabWinHttp : IPlayFabHttp
     {
         private readonly HttpClient _client = new HttpClient();
-        private ISerializerPlugin _serializer;
-
-        public PlayFabSysHttp()
-        {
-            _serializer = (ISerializerPlugin)PluginManager.Instance.GetPlugin(PluginContract.Serializer);
-            if (_serializer == null)
-            {
-                _serializer = new PlayFabSerializer();
-                PluginManager.Instance.SetPlugin(PluginContract.Serializer, _serializer);
-            }
-        }
 
         public async Task<object> DoPost(string urlPath, PlayFabRequestCommon request, string authType, string authKey, Dictionary<string, string> extraHeaders)
```

Comment:
Remove string authType, string authKey.
They'll already have been atdded to extraHeaders before we get here.